Parallel makes are unreliable

Probably not very critical at this point, but `make -j` often fails for me because the top-level rule for making `src/libapachemd.la` overlaps with the work done by the recursive `SUBDIRS`. This leads to errors like

    mv: cannot stat 'acme/.deps/libapachemd_la-md_acme.Tpo': No such file or directory
    Makefile:540: recipe for target 'acme/libapachemd_la-md_acme.lo' failed
    make[1]: *** [acme/libapachemd_la-md_acme.lo] Error 1

At the same time that the top-level Makefile recurses into the `src` subdirectory to start its work (it does this because we've set the `SUBDIRS` variable), the rule for `src/libapachemd.la` is spawning a second process to do the same work with `$(MAKE) -C src`. They step on each other.

--- a/Makefile.am
+++ b/Makefile.am
@@ -20,18 +20,7 @@ ACLOCAL_AMFLAGS = -I m4
 
 dist_doc_DATA   = README README.md LICENSE
 
-.PHONY: all test a2md mod_md libmd
-
-all: a2md mod_md
-
-src/libapachemd.la: src/*.*
-	$(MAKE) -C src
-
-a2md: src/libapachemd.la
-	$(MAKE) -C a2md
-
-mod_md: src/libapachemd.la
-	$(MAKE) -C mod_md
+.PHONY: test
 
 test: a2md/a2md mod_md/mod_md.la
 	$(MAKE) -C test/ test

